craft_application/application.py

```diff
@@ -294,6 +295,20 @@ def run(self) -> int:
             return_code = 128 + signal.SIGINT
         except craft_cli.CraftError as err:
             self._emit_error(err)
+        except craft_parts.PartsError as err:
+            self._emit_error(
+                craft_cli.CraftError(
+                    err.brief, details=err.details, resolution=err.resolution
+                )
+            )
+            return_code = 256
+        except craft_providers.ProviderError as err:
+            self._emit_error(
+                craft_cli.CraftError(
+                    err.brief, details=err.details, resolution=err.resolution
+                )
+            )
+            return_code = 512
```

Comment:
where did these numbers come from?

Reply:
I think I copied them from an early version of craft application from @lengau ... alas I might be making that up!
I'll change it to 1